### PR TITLE
Log levels and CI

### PR DESCRIPTION
**Pull request by glesica**
@smai-f look at all those passing tests and crap like that!

I also made log levels configurable on the command line, although there isn't actually much logging right now other than some info logs to help identify where errors occurred.

### CODE REVIEW

File: pyrs990/_options.py [hunk 1]
```diff
@@ -116,6 +116,14 @@
     help="file to which output should be written (or ':stdout:', ':stderr:')",
 )
 
+parser.add_argument(
+    "--regions",
+    type=str,
+    default="mt",
+    help="regions to search, comma-separated "
+    + "(two character state abbreviations, for most)",
```
Comment (smai-f) [inline pyrs990/_options.py]: What else is there other than state abbreviations? Puerto Rico and DC?

File: README.md [hunk 2]
```diff
@@ -1,31 +1,114 @@
...
     1. `pipenv lock` (updates the lock file)
   1. Make a pull request
 
+## About the Data
+
+Right now we pull data that originated with the IRS (hence the silly name)
+but we get it from a couple sources and information about what is actually
+available is a little spread out as well.
+
+### Structure
```
Comment (smai-f) [inline README.md]: TY!

File: README.md [hunk 3]
```diff
@@ -1,31 +1,114 @@
...
+that must be downloaded a satisfy a given query. The first is an
+annual index (we refer to it as "Annual" or "Annual Index" in the
+code). This index contains all filings processed by the IRS for a
+given calendar year.
+
+Note that this does not necessarily have anything
+to do with the filing year. An organization might, for example, file
+its 2016 990 in either 2017 or 2018 (or even later). There is a field,
+described below, called `tax_period` that reflects the filing period.
+In the future, we intend to further abstract this is that it is
```
Comment (smai-f) [inline README.md]: #nit we intend to further abstract this is that -> we intend to further abstract this so that

File: pyrs990/_options.py (hunk 1 above)
Comment (glesica) [reply pyrs990/_options.py]: International organizations have an abbreviation and then there are regions, like northwest, etc.

**Pull request merged**